src/v/utils/rjson_allocator.h

```diff
@@ -0,0 +1,26 @@
+#include <rapidjson/allocators.h>
+#include <rapidjson/document.h>
+#include <rapidjson/encodings.h>
+
+namespace rjson_allocator {
+class MyAllocator : public rapidjson::CrtAllocator {
```

Comment:
Can this should have a more informative name and be `snake_case`.

I think it would be better to implement concept directly and reuse `CrtAllocator` by composition and delegation rather than inheritance.

Reply:
Althought you may want to use [\[\[no_unique_address\]\] _impl;](https://en.cppreference.com/w/cpp/language/attributes/no_unique_address)